framework/include/utils/MooseArray.h

```diff
@@ -169,6 +169,11 @@ class MooseArray
    */
   std::vector<T> stdVector();
 
+  /**
+   * Reference to first element of array
+   */
+  const T & firstElement() { return *_data; }
```

Comment:
Why not call it `data()` to mirror the vector api

Reply:
👍